src/renderer/components/distraction-settings/distraction-settings.vue

```diff
@@ -99,6 +99,12 @@
           :default-value="hideChannelPodcasts"
           @change="updateHideChannelPodcasts"
         />
+        <ft-toggle-switch
+          :label="$t('Settings.Distraction Free Settings.Hide Channel Home')"
+          :compact="true"
+          :default-value="hideChannelHome"
+          @change="updateHideChannelHome"
+        />
```

Comment:
As the home tab is the first tab on the channel page, could you please move this switch to the start of the channel subsection, so that it somewhat matches the order on the channel page.